skein/test/test_model.py

```diff
@@ -348,7 +348,8 @@ def test_container():
 
     kwargs = dict(service_name="foo",
                   instance=0,
-                  yarn_container_id='container_1528138529205_0038_01_000001')
+                  yarn_container_id='container_1528138529205_0038_01_000001'
+                  yarn_node_address='localhost:14420')
```

Comment:
you're missing a comma above ^^, this is a syntax error.

Reply:
Ouch, fixed.